src/app/redmine.service.ts

```diff
@@ -136,25 +136,25 @@ export class RedmineService {
     momentUnit: string,
     includeFuture: boolean = true,
     onlyWorkingDays: boolean = true,
-    desc: boolean = true
+    desc: boolean = true,
+    includeWholeWeek = false
   ): Observable<DayLog[]> {
     const logMap = [];
-    // debugger;
-
     const period = momentUnit as moment.unitOfTime.StartOf;
     // Using GGGG as ISO year format for week in order to fix https://github.com/moment/moment/pull/4700
     const html5fmt = momentUnit === 'week' ? 'GGGG-[W]WW' : 'YYYY-MM';
 
-    const min = moment(weekOrMonth, html5fmt).startOf(period);
-    const max = moment(weekOrMonth, html5fmt).endOf(period);
+    let min = moment(weekOrMonth, html5fmt).startOf(period);
+    let max = moment(weekOrMonth, html5fmt).endOf(period);
+    if (includeWholeWeek) {
+      min = min.startOf('isoWeek');
```

Comment:
Jinde se pouziva `week`, takze bych preferovala `min.startOf('week')` at je to jednotne. 
V konstruktoru se definuje locale, takze `week` zacina v pondeli.